node/core/pvf/src/prepare/pool.rs

```diff
@@ -321,6 +322,7 @@ fn handle_mux(
 
 					Ok(())
 				},
+				// Sending `rip: true` triggers killing the worker.
```

Comment:
I think this line should be a doc comment on `FromPool::Concluded`

Reply:
Oh, the existing doc on `Concluded` already addresses this, but it says the worker should already be killed:

```rs
/// Indicates whether the worker process was killed.
rip: bool,
```

This happens in `attempt_retire`, which I missed. I had a misunderstanding based on this line:

```rs
// workers are registered upon spawn and removed in one of the following cases:
//   1. received rip signal
//   2. received concluded signal with rip=true;
```

Not sure if we need any comments here, just a brain fart on my end.